Makefile.docker

```diff
@@ -7,6 +7,7 @@ docker-cilium-image-for-developers:
 	# DOCKER_BUILDKIT allows for faster build as well as the ability to use
 	# a dedicated dockerignore file per Dockerfile.
 	$(QUIET)DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE) build \
+		$(DOCKER_FLAGS) \
```

Comment:
Nit: indentation

```suggestion
	     $(DOCKER_FLAGS) \
```

Reply:
We are using tabs in rest of the file, I think I will change identation of the rest of this command instead.